### PR TITLE
Third pull request

### PR DESCRIPTION
**Pull request by Tanrungthip**
## What

1. add create_table
2. add /search
3. add category_id into items table (Join foreign key)
4. adjust /items and /item_id 
## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

File: python/main.py [hunk 1]
```diff
@@ -20,6 +20,37 @@
     allow_headers=["*"],
 )
 
+db_path = "./db/mercari.sqlite3"
+
+def create_table(conn: sqlite3.Connection):
+    table = [
```
Comment (westcottian) [inline python/main.py:26]: It's always advisable to keep the DDL in a separate file.

File: python/main.py [hunk 2]
```diff
@@ -20,6 +20,37 @@
...
+        )
+        """,
+        """
+        CREATE TABLE IF NOT EXISTS category (
+            id INTEGER PRIMARY KEY AUTOINCREMENT,
+            name TEXT NOT NULL
+        )
+        """
+    ]
+    try:
```
Comment (westcottian) [inline python/main.py:43]: It's better if we can keep this in a separate function so that it can be reused.

I'm ok at this moment with this being here.

**Review by westcottian — APPROVED** [review]
LGTM

File: python/main.py (hunk 2 above)
Comment (Tanrungthip) [reply python/main.py:43]: Thank you for the suggestion. I will create a separate function for this

**Pull request merged**